### ISSUE TITLE
can greendao support abs operation ?

### ISSUE THREAD

**Issue by nangonghuang** [greenrobot/greenDAO#658]
I searched the Property class and can't find any method involves abs , like "select * from message where abs(id) > 113 ",  I'm using greendao 3.2.0 . Combination of lt and gt is used at the moment

**Comment by greenrobot-team** [comment]
Yes, this is currently not built-in. You may be able to use a raw SQL query though (`dao.queryRaw()`). -ut
